js/ui/sessionMode.js

```diff
@@ -109,6 +109,40 @@ const _modes = {
             right: ['a11y', 'keyboard', 'aggregateMenu', 'userMenu',
                     'dateMenu', 'hotCorner']
         }
+    },
+
+    'user-coding': {
```

Comment:
Not sure why you added this; those additional components should go in the regular "user" session.